pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -312,6 +312,26 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
   )
   private final Collection<RpcApi> wsApis = null;
 
+  private Long refreshDelay;
+
+  @Option(
+    names = {"--ws-refresh-delay"},
+    paramLabel = "<refresh delay>",
+    arity = "1",
+    description =
+        "Refresh delay of websocket subscription sync in milliseconds. "
+            + "default: ${DEFAULT-VALUE}",
+    defaultValue = "5000"
+  )
+  private void setRefreshDelay(final Long refreshDelay) {
```

Comment:
what happens if I specify 1.5 ? Why long not int?

Reply:
vertx handles the sync function after we pass it in a value and it requires a long.
If 1.5 is entered, the CLI will pick it up as an error and warn user because only positive int below 3600000 is accepted.
long datatype are integers, they cannot take in decimals unlike double precision numbers